x-pack/plugins/apm/public/selectors/chartSelectors.ts

```diff
@@ -31,40 +31,37 @@ export interface ITpmBucket {
 }
 
 export interface ITransactionChartData {
-  tpmSeries: ITpmBucket[];
```

Comment:
I'd love to see if we can get rid of this file. Not sure if it's possible but it feels like there's a lot of extra data massaging going on that we don't need.